web/impact/impact/graphql/types/entrepreneur_profile_type.py

```diff
@@ -49,7 +49,8 @@ def resolve_title(self, info, **kwargs):
 
     def resolve_program_roles(self, info, **kwargs):
```

Comment:
I notice that these functions are duplicates of each other. Can we possible refactor out the duplication?